x-pack/platform/plugins/shared/stack_connectors/common/index.ts

```diff
@@ -10,6 +10,7 @@ export enum AdditionalEmailServices {
   ELASTIC_CLOUD = 'elastic_cloud',
   EXCHANGE = 'exchange_server',
   OTHER = 'other',
+  AWS_SES = 'ses',
```

Comment:
nit: It seems that this enum is only for services that are not recognized by the nodemailer. Should we use another enum?

Reply:
I checked and I was secure to use it. I think the name is misleading